frontend/compose.test.yml

```
@@ -1,13 +1,30 @@
 name: damnit-web-test
 
 services:
+  site:
+    build:
+      dockerfile: ./apps/site/Dockerfile
+      target: build
+      args:
+        APP_URL: https://damnit-test.xfel.eu/app
+    image: damnit-web-site:test
+
+  app:
+    build:
+      dockerfile: ./apps/app/Dockerfile
+      target: build
+      args:
+        SITE_URL: https://damnit-test.xfel.eu
+        BASE_PATH: ${APP_BASE_PATH?}
+    image: damnit-web-app:test
+
   frontend:
     build:
-      context: .
-      target: production
+      dockerfile: ./Dockerfile
     environment:
```

Should add a dependency to the other services:

```suggestion
  frontend:
    build:
      dockerfile: ./Dockerfile
    depends_on:
        - site
        - app
    environment:
```